Modules/@babylonjs/react-native-windows/package.json

```diff
@@ -24,7 +24,7 @@
   "licenseFilename": "LICENSE",
   "readmeFilename": "README.md",
   "peerDependencies": {
-    "@babylonjs/core": "^5.2.0",
```

Comment:
Looks like the package-lock.json changes associated with this change are missing.